7.0.x-qa-30288

Hanlf opened the pull request:
https://issues.liferay.com/browse/LRQA-30288
ee-7.0.x only 
CC @GinsonRen

brianwulbern commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/40989
https://github.com/brianchandotcom/liferay-portal/pull/40990

rodrigopaulino committed:
LPS-69224 Fix for comments on blogs while impersonating user

ksuetani committed:
LRQA-28149 Update app manager app name path

ksuetani committed:
LRQA-28149 Refine next page link path to not include disabled link

ksuetani committed:
LRQA-28149 Add macro to select apps and verify app is not present on multiple pages in app manager

ksuetani committed:
LRQA-28149 Add new test to verify Independent Modules is not nested

Commit:
LPS-69224 SF

adolfopa committed:
LPS-70180 Initialize context variables in the `init` method

Also, defer returning those values until we have the correct
locale. The impl before the fix would always use the current user
locale instead of the locale of the target user.

adolfopa committed:
LPS-70180 Semver

adolfopa committed:
LPS-70180 Fix variables in templates

adolfopa committed:
LPS-70180 Keep baseline happy

There may be some problem with the baseline tool. It is detecting API
changes where there seems to be none. In this particular case:

  - The superclass (SubscriptionSender) has (among others) a public
    `void initialize(void)` and a protected `String
    replaceContent(String, Locale)` methods. Obviously, both methods
    are part of the subclass API.

  - The subclass overrides the `String replaceContent(String, Locale)`
    method.

After moving some code around, the situation changed to the following:

  - The subclass overrides the `void initialize(void)` method.

  - The subclass no longer overrides the `replaceContent` method.

No changes to the superclass were made, and the set of methods exposed
by the subclass is exactly the same as before (i.e. no API changes),
but `gw baseline` is complaining about major, incompatible changes.

The only purpose of this commit is to keep backwards compatibility and
also to serve as an example in case there really is a problem with the
baseline tool.

Commit:
LPS-70180 Leave the additional square brackets to delimit the site and the category

codyhoag committed:
LPS-70170 Use verb at beginning of changelogs, when possible. I think we can assume that this is "always" done based on the wording.

xbrianlee committed:
LRQA-29362 Navigate back to Roles portlet before adding another role

alee8888 committed:
LRQA-28149 SF

Hanlf committed:
LRQA-30288 Update the var to match ee-7.0.x

liferay-continuous-integration commented:
<h1>Some tests FAILED.</h1><p>Build Time: 10 minutes 56 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/7.0.x">7.0.x</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/395696ec0fc14e0870f5e2b0b3d0b1f151732a8f">395696ec0fc14e0870f5e2b0b3d0b1f151732a8f</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/81/">test-portal-acceptance-pullrequest(7.0.x)</a></strong></strike></li><li><strike><strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-dist(7.0.x)/66/">test-portal-acceptance-pullrequest-dist(7.0.x)/jdk8</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-source(7.0.x)/105/">test-portal-acceptance-pullrequest-source(7.0.x)</a></li></ul><h5>For more details click <a href="https://test-1-5.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(7.0.x)/builds/81/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/81/">test-portal-acceptance-pullrequest(7.0.x)</a></h5><h6>Job Results:</h6><p>1 Job Passed.<br />2 Jobs Failed.</p><pre><code>Downstream jobs FAILED</code></pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-dist(7.0.x)/66/">test-portal-acceptance-pullrequest-dist(7.0.x)/jdk8</a></h5><h6>Job Results:</h6><p>0 Tests Passed.<br />4 Tests Failed.</p><ol><li><strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-dist%287.0.x%29/AXIS_VARIABLE=jboss,label_exp=!master/66/">test-portal-acceptance-pullrequest-dist&#40;7.0.x&#41; ?? jboss,!master #66</a></strong><pre><code>     [exec] :apps:knowledge-base:knowledge-base-web:copyLibs SKIPPED
     [exec] :apps:knowledge-base:knowledge-base-web:replaceSoyTranslation UP-TO-DATE
     [exec] :apps:knowledge-base:knowledge-base-web:classes
     [exec] :apps:knowledge-base:knowledge-base-web:jar
     [exec] :apps:mail-reader:mail-reader-service:deploy
     [exec] :apps:mail-reader:mail-reader-service:deployDependencies UP-TO-DATE
     [exec] :apps:marketplace:marketplace-api:compileJava
     [exec] Parsed /META-INF/resources/css/main.scss in 985ms
     [exec] :apps:knowledge-base:knowledge-base-service:compileJava FAILED
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 3 errors
     [exec] Restoring node_modules of project ':apps:frontend-theme-westeros-bank:frontend-theme-westeros-bank' from /root/.liferay/node-modules-cache/520724501
     [exec] Restoring node_modules of project ':apps:frontend-theme-porygon:frontend-theme-porygon' from /root/.liferay/node-modules-cache/-2014623178
     [exec] Restoring node_modules of project ':apps:frontend-theme-fjord:frontend-theme-fjord' from /root/.liferay/node-modules-cache/520724501
     [exec] 
     [exec] BUILD FAILED
     [exec] 
     [exec] Total time: 1 mins 13.976 secs
     [exec] Gradle build finished at 2017-01-19 07:20:44.301
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] > Compilation failed; see the compiler error output for details.
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)</code></pre></li><li><strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-dist%287.0.x%29/AXIS_VARIABLE=tomcat,label_exp=!master/66/">test-portal-acceptance-pullrequest-dist&#40;7.0.x&#41; ?? tomcat,!master #66</a></strong><pre><code>     [exec] Parsed /META-INF/resources/display/css/main.scss in 5ms
     [exec] Parsed /META-INF/resources/section/css/main.scss in 4ms
     [exec] :apps:knowledge-base:knowledge-base-web:processResources
     [exec] Using native Sass compiler
     [exec] :apps:knowledge-base:knowledge-base-web:transpileJS SKIPPED
     [exec] :apps:knowledge-base:knowledge-base-web:configJSModules SKIPPED
     [exec] :apps:knowledge-base:knowledge-base-web:copyLibs SKIPPED
     [exec] :apps:knowledge-base:knowledge-base-web:replaceSoyTranslation UP-TO-DATE
     [exec] :apps:knowledge-base:knowledge-base-web:classes
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 3 errors
     [exec] :apps:knowledge-base:knowledge-base-web:jar
     [exec] :apps:knowledge-base:knowledge-base-service:compileJava FAILED
     [exec] Parsed /META-INF/resources/css/main.scss in 722ms
     [exec] Restoring node_modules of project ':apps:frontend-theme-westeros-bank:frontend-theme-westeros-bank' from /root/.liferay/node-modules-cache/520724501
     [exec] Restoring node_modules of project ':apps:frontend-theme-porygon:frontend-theme-porygon' from /root/.liferay/node-modules-cache/-2014623178
     [exec] Restoring node_modules of project ':apps:frontend-theme-fjord:frontend-theme-fjord' from /root/.liferay/node-modules-cache/520724501
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] > Compilation failed; see the compiler error output for details.
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)</code></pre></li><li><strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-dist%287.0.x%29/AXIS_VARIABLE=weblogic,label_exp=!master/66/">test-portal-acceptance-pullrequest-dist&#40;7.0.x&#41; ?? weblogic,!master #66</a></strong><pre><code>     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 3 errors
     [exec] :apps:marketplace:marketplace-api:compileJava
     [exec] Parsed /META-INF/resources/css/main.scss in 848ms
     [exec] :apps:marketplace:marketplace-api:buildCSS UP-TO-DATE
     [exec] :apps:marketplace:marketplace-api:processResources
     [exec] :apps:marketplace:marketplace-api:transpileJS SKIPPED
     [exec] :apps:marketplace:marketplace-api:configJSModules SKIPPED
     [exec] :apps:marketplace:marketplace-api:copyLibs SKIPPED
     [exec] :apps:marketplace:marketplace-api:replaceSoyTranslation UP-TO-DATE
     [exec] :apps:marketplace:marketplace-api:classes
     [exec] :apps:marketplace:marketplace-api:jar
     [exec] :apps:knowledge-base:knowledge-base-service:compileJava FAILED
     [exec] Restoring node_modules of project ':apps:frontend-theme-westeros-bank:frontend-theme-westeros-bank' from /root/.liferay/node-modules-cache/520724501
     [exec] Restoring node_modules of project ':apps:frontend-theme-porygon:frontend-theme-porygon' from /root/.liferay/node-modules-cache/-2014623178
     [exec] Restoring node_modules of project ':apps:frontend-theme-fjord:frontend-theme-fjord' from /root/.liferay/node-modules-cache/520724501
     [exec] 
     [exec] BUILD FAILED
     [exec] 
     [exec] Total time: 1 mins 16.412 secs
     [exec] Gradle build finished at 2017-01-19 07:20:52.087
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] > Compilation failed; see the compiler error output for details.
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:knowledge-base:knowledge-base-service:compileJava'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)</code></pre></li></ol></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(7.0.x)/">here</a>.</h5>

Hanlf commented:
ci:retest

liferay-continuous-integration commented:
<h1>Some tests FAILED.</h1><p>Build Time: 1 hour 3 minutes 9 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/7.0.x">7.0.x</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9a89f39359ee23cab940a653d472deb0bf7221b5">9a89f39359ee23cab940a653d472deb0bf7221b5</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/84/">test-portal-acceptance-pullrequest(7.0.x)</a></strong></strike></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3915/">test-portal-acceptance-pullrequest-batch(7.0.x)/compile-jsp-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3512/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4035/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-db2105-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3953/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2245/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3854/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3925/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3576/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat80-sybase160-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3753/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3845/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3657/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4022/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3892/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4036/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/10</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4023/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/11</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3893/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/12</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3754/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/13</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3846/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/14</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4024/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/15</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1955/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/16</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3800/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/17</a></li><li><strike><strong><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3959/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/18</a></strong></strike></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3820/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/19</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3658/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3927/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/20</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3917/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/21</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3578/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/22</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3640/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/23</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4037/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/24</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3982/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/25</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3856/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/26</a></li><li><strike><strong><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/908/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/27</a></strong></strike></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2247/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/28</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3847/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/29</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3855/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3659/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/30</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3755/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/31</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3918/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/32</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2213/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/33</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3894/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/34</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3983/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/35</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3639/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/36</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3928/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/37</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4038/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/38</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/4025/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/39</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3916/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/4</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3801/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/40</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3956/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/41</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3756/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/42</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3848/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/43</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3857/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/44</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3579/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/45</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2246/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/5</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3577/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/6</a></li><li><strike><strong><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3515/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/7</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3926/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3954/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/9</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3858/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3919/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-jboss70-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3514/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3661/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3849/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3822/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3757/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1957/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3920/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3581/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3859/">test-portal-acceptance-pullrequest-batch(7.0.x)/portal-web-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3958/">test-portal-acceptance-pullrequest-batch(7.0.x)/source-format-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-dist(7.0.x)/51/">test-portal-acceptance-pullrequest-dist(7.0.x)/jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-source(7.0.x)/80/">test-portal-acceptance-pullrequest-source(7.0.x)</a></li></ul><h5>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(7.0.x)/builds/84/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/84/">test-portal-acceptance-pullrequest(7.0.x)</a></h5><h6>Job Results:</h6><p>68 Jobs Passed.<br />4 Jobs Failed.</p><pre><code>[beanshell] TOP_LEVEL_MASTER_HOSTNAME=test-1-6
[beanshell] BUILD_DESCRIPTION=<strong>brianwulbern</strong> - <a href="https://github.com/brianwulbern/liferay-portal/pull/170">PR#170</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(7.0.x)/builds/84/jenkins-report.html">Jenkins Report</a>
[beanshell] TOP_LEVEL_BUILD_URL=https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/84/
[beanshell] TOP_LEVEL_BRANCH_NAME=7.0.x-temp-20170121223414681
[beanshell] JENKINS_GITHUB_UPSTREAM_BRANCH_NAME=master
[beanshell] GITHUB_SENDER_USERNAME=Hanlf
[beanshell] JENKINS_GITHUB_RECEIVER_USERNAME=liferay
[beanshell] TOP_LEVEL_BUILD_NUMBER=84
[beanshell] DIST_ROOT_PATH=/test-1-6
[beanshell] TOP_LEVEL_START_TIME=1485066855263
[beanshell] JENKINS_GITHUB_PULL_REQUEST_NUMBER=0
[beanshell] TOP_LEVEL_BUILD_DESCRIPTION=<strong>brianwulbern</strong> - <a href="https://github.com/brianwulbern/liferay-portal/pull/170">PR#170</a> - <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(7.0.x)/builds/84/jenkins-report.html">Jenkins Report</a>
[beanshell] 
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/json.data
     [echo] curl -d @/opt/dev/projects/github/liferay-jenkins-ee/json.data -H "Content-Type: application/json" --silent --output /dev/null --write-out %{http_code} -X POST http://cloud-10-0-0-31/osb-jenkins-web/map/test-1-6/test-portal-acceptance-pullrequest%287.0.x%29/84/test-portal-acceptance-pullrequest-batch%287.0.x%29/functional-upgrade-tomcat80-mysql56-jdk8/0/start.properties
     [exec] 200
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20170121225600907
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/json.data
     [echo] curl --output /opt/dev/projects/github/liferay-jenkins-ee/json.data --silent --write-out %{http_code} http://cloud-10-0-0-31/osb-jenkins-web/map/test-1-6/test-portal-acceptance-pullrequest%287.0.x%29/84/test-portal-acceptance-pullrequest-batch%287.0.x%29/functional-upgrade-tomcat80-mysql56-jdk8/0/start.properties
     [exec] 200
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20170121225600917
[beanshell] ##
[beanshell] ## /tmp/jenkins/test-1-6/test-portal-acceptance-pullrequest(7.0.x)/84/test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat80-mysql56-jdk8/0/start.properties
[beanshell] ##
[beanshell] </code></pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3959/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/18</a></h5><h6>Job Results:</h6><p>1 Test Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch%287.0.x%29/AXIS_VARIABLE=0,label_exp=!master/3959//testReport/com.liferay.poshi.runner/PoshiRunner/test_Calendar_DeleteEventRegular_">Calendar#DeleteEventRegular</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/18/0/Calendar_DeleteEventRegular/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/18/0/Calendar_DeleteEventRegular/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/18/0/jenkins-console.txt.gz">Console Output</a></li></ol></li><li><h5 job-result="FAILURE"><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/908/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/27</a></h5><h6>Job Results:</h6><p>7 Tests Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch%287.0.x%29/AXIS_VARIABLE=6,label_exp=!master/908/">test-portal-acceptance-pullrequest-batch&#40;7.0.x&#41; ?? 6,!master #908</a></strong><pre><code>clean-up-web-xml:
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal-7.0.x/bundles/tomcat-8.0.32/webapps/ROOT/WEB-INF
     [exec] Stopping mysqld:  [  OK  ]
     [echo] 
      [get] Getting: http://test-1-21/job/test-portal-acceptance-pullrequest-batch(7.0.x)/AXIS_VARIABLE=6,label_exp=!master/908//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal-7.0.x/20170122072835845.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal-7.0.x/20170122072835845.txt</code></pre></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/3515/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat80-mysql56-jdk8/7</a></h5><h6>Job Results:</h6><p>9 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch%287.0.x%29/AXIS_VARIABLE=0,label_exp=!master/3515//testReport/com.liferay.poshi.runner/PoshiRunner/test_Calendar_AddEventAllDay_">Calendar#AddEventAllDay</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/7/0/Calendar_AddEventAllDay/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/7/0/Calendar_AddEventAllDay/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-6/1485066855263/test-portal-acceptance-pullrequest(7.0.x)/84/functional-tomcat80-mysql56-jdk8/7/0/jenkins-console.txt.gz">Console Output</a></li></ol></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(7.0.x)/">here</a>.</h5>

brianwulbern commented:
No test changes needed :)

brianwulbern commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/46713
